tests/components/samsungtv/test_config_flow.py

```diff
@@ -540,13 +497,16 @@ async def test_ssdp(hass: HomeAssistant) -> None:
 @pytest.mark.usefixtures("remote", "rest_api_failing")
 async def test_ssdp_no_manufacturer(hass: HomeAssistant) -> None:
     """Test starting a flow from discovery when the manufacturer data is missing."""
+    ssdp_data = deepcopy(MOCK_SSDP_DATA)
+    ssdp_data.upnp.pop(ATTR_UPNP_MANUFACTURER)
+
```

Comment:
Since we don't have a "real-life" fixture, I think it is clearer to re-use the default fixture and tweak inside the test